### DISCUSSION TITLE
Following feed bug

### DISCUSSION THREAD

**Discussion by jeesusonherraturku** [community/community#44921] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

There is a person that I followed (he followed first) and I unfollowed who still occupies my Following feed, and has been for 2 months. 

It reads: "personX started following personY and 298 other users • 1 min"

(real username replaced with personXY)

This same text and number appear in my feed for 2 months although he has thousands of followers. The number 298 does not seem to change. I would like this feed to disappear since I have no idea who he is, and I'm not part of any project he is etc. 

I would prefer to choose myself who appear on my Following list. Thank you!

**Comment by KingWebs** [comment]
Check your notifications and inscribe from that account.

**Comment by jeesusonherraturku** [comment]
I see no mention of that person anywhere in my notifications of subscriptions etc... I took a picture of the problem. 

![Screenshot_20230121_191444](https://user-images.githubusercontent.com/117400574/213878804-7e12832e-a9ce-400d-b752-d955cbf71854.jpg)

**Comment by Ay-iman** [comment]
check very well is he among the followers or following he might have been remove among the following

**Comment by jeesusonherraturku** [comment]
I checked that I am not following anyone  nor is anyone following me. I checked with both android chrome and Android Firefox.

**Comment by jeesusonherraturku** [comment]
I blocked the user and that helped. That is a temporary solution to the problem.
